packages/theme/components/ConfigurableProductDetails.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <SfSelect v-model="size" @change="handleChange" label="Size" class="sf-select--bordered product-details__attribute">
+      <SfSelectOption v-for="size in possibleOptions.sizes" :key="size.value" :value="size.value">
+        <SfProductOption :label="size.label" />
+      </SfSelectOption>
+    </SfSelect>
+    <SfSelect v-model="color" @change="handleChange" label="Color" class="sf-select--bordered product-details__attribute">
+      <SfSelectOption v-for="color in possibleOptions.colors" :key="color.value" :value="color.value">
+        <SfProductOption :label="color.label" :color="color.color" />
+      </SfSelectOption>
+    </SfSelect>
+  </div>
+</template>
+
+<script>
+import {
+  SfSelect,
+  SfProductOption
+} from "@storefront-ui/vue";
+import { computed, ref, watch } from '@vue/composition-api'
+
+export default {
+  props: {
+    productOptions: Array,
+  },
+  setup ({ productOptions }, { emit }) {
+    const possibleOptions = computed(() => {
+      const sizeOptions = productOptions.find(o => o.attributeName === 'size')
```

Comment:
co jesli jest wiecej niz size i color? Musimy to miec jakos zautmatyzowane najlepiej 1 propertiesem